Cargo.lock

```diff
@@ -15,6 +15,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 name = "api_server"
 version = "0.1.0"
 dependencies = [
+ "arch 0.1.0",
```

Comment:
Unless someone can contribute a better name, I think `arch` is the right way to go. This is a standard naming convention in many projects.

IMHO renaming would only be necessary if we were to make this crate public. But in the context of an internal crate containing various arch-specific code, `arch` is a perfectly valid name.

Reply:
I **really** don't like that name because it doesn't say anything about the content of the crate. It is just telling me that the code there might work on many architectures or be specific to some architectures. I am not saying we should change it now, but just to be open about changing it and maybe refactor the contents of it at a later time.

Reply:
`arch-specific-misc-code` 😆